### ISSUE TITLE
Accessibility: Headings for widgets should not necessarily be H3

### ISSUE THREAD

**Issue by caaster** [projectblacklight/spotlight#3473]
### Issue Description - WCAG standard
WCAG heading markup rules require that headings, including main page titles and section titles, are marked up using semantic HTML heading elements (h1-h6). These headings must be presented in a logical order, reflecting the content's hierarchy.

Any Spotlight widget that has the ability to create a heading does so as a H3. There are many use cases where this works just fine, because there is an H2 heading on the page (that makes sense for the content) above the H3. However, there are just as many cases where there is appropriately NO H2 heading on the page, and so the jump to H3 is incorrect for accessibility reasons.

### Method of discovery
Stanford Office of Digital Accessibility review of Exhibits, Stanford's Spotlight instance.

### Steps to Reproduce
On any Spotlight home, feature, or about page, create some content, including a widget that contains a heading. Then you can see the H3 in the markup.

### Suggested solution
Ideally there will be a solution that doesn't require massive "by hand" remediation of already published exhibits.
